.github/workflows/ci.yaml

```diff
@@ -0,0 +1,55 @@
+# This is a basic workflow
+
+name: ci
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Use Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      # Run install dependencies
+      - name: Install dependencies
+        run: |
+          pipx install pre-commit
```

Comment:
I prefer seeing a distinct log entry for this. Also, since it's not used anywhere, no need to integrate it in GHA prematurely.

Reply:
Removed